shared/templates/kernel_build_config/oval.template

```diff
@@ -0,0 +1,30 @@
+<def-group>
+  <definition class="compliance" id="{{{ _RULE_ID }}}"
+  version="1">
+    {{{ oval_metadata("The kernel " + CONFIG + " should have value" + VALUE, affected_platforms=["multi_platform_all"]) }}}
+    <criteria operator="OR">
+      <criterion comment="Check presence of build configuration of installed kernels" test_ref="test_kernel_{{{ CONFIG | lower }}}" />
+{{% if SHOULD_CHECK_ABSENCE %}}
+      <criterion comment="Check absense of build configuration of installed kernels" test_ref="test_kernel_{{{ CONFIG | lower }}}_absence" />
+{{% endif %}}
+    </criteria>
+  </definition>
+
+  <ind:textfilecontent54_test check="all" comment="Check /boot/config-.* files for {{{ CONFIG }}}={{{ VALUE }}}"
+  id="test_kernel_{{{ CONFIG | lower }}}" version="1">
+    <ind:object object_ref="object_kernel_{{{ CONFIG | lower }}}" />
+  </ind:textfilecontent54_test>
+
+  <ind:textfilecontent54_object id="object_kernel_{{{ CONFIG | lower }}}" version="1">
+    <ind:filepath operation="pattern match">^/boot/config-.*$</ind:filepath>
+    <ind:pattern operation="pattern match">^{{{ CONFIG }}}="?{{{ VALUE }}}"?$</ind:pattern>
+    <ind:instance datatype="int" operation="greater than or equal">1</ind:instance>
+  </ind:textfilecontent54_object>
+
+{{% if SHOULD_CHECK_ABSENCE %}}
+  <ind:textfilecontent54_test check="all" check_existence="none_exist" comment="Check /boot/config-.* files for {{{ CONFIG }}}={{{ VALUE }}}"
+  id="test_kernel_{{{ CONFIG | lower }}}_absence" version="1">
+    <ind:object object_ref="object_kernel_{{{ CONFIG | lower }}}" />
+  </ind:textfilecontent54_test>
```

Comment:
This test should check if a parameter is not configured in any way. You can't reuse the object above since it looks for the whole line `kernel_config=n` for example.

This test needs to check if the `config` is absent/commented out.

I believe this was not manifesting before because of the `shoud` typo.